src/Compilers/CSharp/Test/Semantic/Semantics/NullableReferenceTypesTests.cs

```diff
@@ -24045,6 +24069,60 @@ void Init()
                 );
         }
 
+        [Fact]
+        public void ConstructorUsesStateFromInitializers()
+        {
+            var source = @"
+public class Program
+{
+    public string Prop { get; set; } = null!;
```

Comment:
>null [](start = 39, length = 4)

Consider using a constant value that is easily visible as non-null.

Reply:
Done